flaskblog/posts/routes.py

```diff
@@ -0,0 +1,65 @@
+from flask import render_template, url_for, flash, redirect, request, abort, Blueprint
+from flask_login import current_user, login_required
+from flaskblog import db
+from flaskblog.models import Post
+from flaskblog.posts.forms import PostForm
+
+
+posts = Blueprint("posts", __name__, template_folder="templates")
+
+
+@posts.route("/post/new", methods=["GET", "POST"])
```

Comment:
Is this post functionality from some tutorial?
I don't know what it has to do with your bookings app.